examples/readme-example.ipynb

```diff
@@ -182,7 +182,7 @@
    "source": [
     "# Set up a sampling routine with 3 MCMC chains, all starting from the same point\n",
     "mcmc = pints.MCMCController(\n",
-    "    log_likelihood, 3, [initial_point]*3, method=pints.AdaptiveCovarianceMCMC)\n",
+    "    log_likelihood, 3, [initial_point]*3, method=pints.RemiACMCMC)\n",
```

Comment:
Happy to use a different name options are:

- RemiACMCMC
- BardenetACMCMC
- RemiACMC
- BardenetACMC -- my favourite!

@martinjrobins @MichaelClerx -- which of these is yours? If you let me know, I can change it over quickly.

Reply:
Happy with BardenetACMC too!

Reply:
@mirams any opinion on this? Ben's rewriting the bit of PINTS that has our trusty AdaptiveCovarianceMCMC method in it, because there's several adaptive covariance methods out there. Remi's one will still be the default, but needs a more specific name

Reply:
What's the difference between remi and the other adaptive covariance one?